client/src/components/FlightAirbuddies.vue

```diff
@@ -28,6 +28,7 @@ const onShowAirbuddies = async () => {
   }
 };
 
+// TODO: Maybe clean DB from obsolete "n/a" strings
```

Comment:
Das hat nie in der DB drin gestanden; Das wurde nur vom automatisch vom Legacy Service ergänzt, wenn es keinen Prozentwert gibt

Reply:
ah jetzt hab Ichs verdtanden